atomic-structure/content.md

```diff
@@ -0,0 +1,25 @@
+# Radioactive Decay
+#### Learning objective
+Basic understanding of atomic structure
```

Comment:
My personal preference is for learning objectives to be full sentences that  use an action verb as the main verb (perhaps in the future tense). This allows them to be more measurable. The action verb should be associated with a Bloom's taxonomy level, like those in this table:
![table](https://s-media-cache-ak0.pinimg.com/originals/cc/6c/4d/cc6c4dd4668e8223d2334b7cc6886b29.jpg)

In this case, maybe something like "The student will be able to reproduce a drawing of the shell model of the atom." would be a learning objective in the "remember" level. An "understanding" level learning objective might be "The student will be able to describe the shell model of the atom".

See examples here :
 [https://tips.uark.edu/using-blooms-taxonomy](https://tips.uark.edu/using-blooms-taxonomy/)